README.md

```diff
@@ -721,6 +722,20 @@ black statick statick_tool tests
 docformatter -i --wrap-summaries 88 --wrap-descriptions 88 <file>
 ```
 
+## Third Party Plugins
+
+Known third party Statick plugins.
+
+| Plugin Name      | Repository Location                              |
+| :--------------- | :----------------------------------------------- |
+| statick-action   | <https://github.com/sscpac/statick-action>       |
```

Comment:
i don't think i would call this a plugin, that seems to imply that it will lint github action files to me. I'd add another section calling out that we have a github action. I think it makes sense to put that in an "Installation" section which I've seen other software do (like mega-linter: https://megalinter.github.io/latest/installation/)